administrator/components/com_admin/sql/updates/postgresql/6.0.0-2025-10-11.sql

```diff
@@ -1,2 +1,5 @@
 ALTER TABLE "#__history"
-    ADD COLUMN "is_current" SMALLINT NOT NULL DEFAULT 0 /** CAN FAIL **/;
\ No newline at end of file
+    ADD COLUMN "is_current" SMALLINT NOT NULL DEFAULT 0 /** CAN FAIL **/;
+ALTER TABLE "#__history"
+    ADD COLUMN "is_legacy" SMALLINT NOT NULL DEFAULT 0 /** CAN FAIL **/;
+UPDATE "#__history" SET "is_legacy" = 1;
```

Comment:
```suggestion
UPDATE "#__history" SET "is_legacy" = 1;

```
Have a new line at the end of the file.